lib/scala/filewatcher/src/main/scala/org/enso/filewatcher/WatcherAdapter.scala

```diff
@@ -1,9 +1,8 @@
-package org.enso.languageserver.filemanager
-
-import java.nio.file.Path
+package org.enso.filewatcher
```

Comment:
Does this mean there already was a working file watcher in the system? All those [deeply thought ideas](https://github.com/enso-org/enso/issues/7178#issuecomment-1634588128) how to implemented are in vain? We are about to use some `io.methvin.watch` library instead?

Please write that info into the #7178 issue.